bower.json

```diff
@@ -19,13 +18,13 @@
     "paper-fab": "polymerelements/paper-fab#1 - 2",
     "paper-icon-button": "polymerelements/paper-icon-button#1 - 2",
     "iron-input": "polymerelements/iron-input#1 - 2",
-    "iron-autogrow-textarea": "polymerelements/iron-autogrow-textarea#1 - 2"
+    "iron-autogrow-textarea": "polymerelements/iron-autogrow-textarea#1 - 2",
+    "web-animations-js": "web-animations/web-animations-js#^2.2.0"
```

Comment:
Actually I don't include the import. this just makes it available for the codelab for convenience

See:
https://codelabs.developers.google.com/codelabs/polymer-firebase-pwa/#8